### PR TITLE
U/viviana/tile css and clear tiles from rack

### PR DESCRIPTION
**Pull request by viviana-aline**

### CODE REVIEW

**Comment by viviana-aline** [comment]
image

File: src/App.tsx [hunk 1]
```diff
@@ -5,12 +5,14 @@ import Board from "./components/board";
...
 import { Rack } from "./components/rack";
 import CompleteTurn from "./components/completeTurn";
+import "./styles/variables.scss";
 import "./styles/reset.scss";
 
 const PLAYERS = 2;
 const INITIAL_CELLS = Array(15)
   .fill(null)
   .map(() => Array(15).fill(null));
+const tilesOnRack = ["A", "B", "C", "D", "E", "F", "G"];
```
Comment (drewrodrigues) [inline src/App.tsx]: Needs to be state if it updates the UI

File: src/App.tsx [hunk 2]
```diff
@@ -21,28 +23,29 @@ interface CurrentTurn {
...
+      setCurrentlySelectedTile(undefined);
+      tilesOnRack.splice(currentlySelectedTile, 1);
     } else {
       console.log("No tile selected");
     }
   }
 
-  function onTileSelect(selectedTile: string) {
-    setcurrentlySelectedTile(selectedTile);
+  function onTileSelect(selectedTile: number) {
```
Comment (drewrodrigues) [inline src/App.tsx]: `selectedTileIndex`?

File: src/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// Game Colors
```
Comment (drewrodrigues) [inline src/styles/variables.scss:1]: Remove variables that are not re-used

File: src/components/cell.tsx [hunk 4]
```diff
@@ -14,12 +14,14 @@ export default function Cell({ row, col, letter, onClick }: CellProps) {
   const bonus = BONUS_CELLS[row][col];
   const bonusText = bonus && bonus.replace("-", " ");
   const isCenter = row === 7 && col === 7;
+  const hasLetter = letter !== null;
```
Comment (drewrodrigues) [inline src/components/cell.tsx]: `isFilled` or change the class to `--has-letter`?

**Review by drewrodrigues — CHANGES_REQUESTED** [review]

**Review by drewrodrigues — APPROVED** [review]
🔥  🚒  👨🏿‍🚒

**Pull request merged**